src/tests/ftest/util/daos_utils.py

```diff
@@ -81,7 +81,12 @@ class DaosCommand(DaosCommandBase):
         "container_list_attrs": r"\n \S+ \S+ (.+)",
         # Sample create-snap output.
         # snapshot/epoch 1582610056530034697 has been created
-        "container_create_snap": r"[A-Za-z\/]+\s[0-9]+\s[a-z\s]+"
+        "container_create_snap": r"[A-Za-z\/]+\s([0-9]+)\s[a-z\s]+",
+        # Sample container list-snaps output - no line break on second line.
+        # 05/15-20:40:40.46 wolf-3 Container's snapshots :
+        # 05/15-20:40:40.46 wolf-3 1589575199653691408  05/15-20:40:40.46
+        # wolf-3 1589575236150689813  05/15-20:40:40.46 wolf-3
+        "container_list_snaps": r"(\d{19})"
```

Comment:
As we add new methods we shouldn't be making use of the deprecated `get_output` method and this output parsing should be moved inside of the new method being added.

Reply:
Done